vassal-app/src/main/java/VASSAL/counters/BoundsTracker.java

```diff
@@ -17,10 +17,12 @@
  */
 package VASSAL.counters;
 
-import java.util.HashSet;
-import java.util.Set;
+import java.awt.Rectangle;
+import java.util.List;
+import java.util.LinkedList;
 
 import VASSAL.build.module.Map;
+import VASSAL.tools.lang.Pair;
```

Comment:
Most places we've switched over to `org.apache.commons.lang3.tuple.Pair`.

Reply:
@uckelman Did you not switch everywhere because modules can rely on existing code that uses `VASSAL.tools.lang.Pair`?

Reply:
This was a while ago now, so my memory is a bit fuzzy---but I think I switched out all the purely internal usage of `VASSAL.tools.lang.Pair` and left the ones which are in signatures of public methods. If I'd been expecting a _lot_ more longevity with V3 codebase than I am, I would likely have deprecated `VASSAL.tools.lang.Pair` as well and added some methods with signatures using `org.apache.commons.lang3.tuple.Pair`.